pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -286,6 +289,42 @@ func newVSphere(cfg VSphereConfig) (*VSphere, error) {
 	return &vs, nil
 }
 
+// Reads vSphere configuration from system environment and construct vSphere object 
+func GetVSphere() (*VSphere, error) {
```

Comment:
Can we create a util.go file and move this there since this code is not really used by the core cloud provider.